Assets/MRTK/SDK/Experimental/NonNativeKeyboard/Scripts/NonNativeKeyboard.cs

```diff
@@ -628,6 +640,7 @@ public void AppendValue(KeyboardValueKey valueKey)
         public void FunctionKey(KeyboardKeyFunc functionKey)
         {
             IndicateActivity();
+            OnKeyboardKeyFuncPressed(functionKey);
```

Comment:
The order here means that when the event is fired, the internal state isn't updated yet right? i.e. if I ask for IsShifted, this will not yet be updated.

We can go either way, but we should document in the comments/remarks for what the expectations are.

Reply:
Done, good point. Thanks Will for the review!